src/services/ehr-repo/get-ehr.js

```diff
@@ -1,49 +1,31 @@
-import { initializeConfig } from "../../config";
+import { config } from "../../config";
 import axios from "axios";
-import { logError, logInfo } from "../../middleware/logging";
-import { EhrUrlNotFoundError, EhrDownloadError } from "../../errors/errors";
+import { logInfo, logError } from "../../middleware/logging";
+import { downloadFromUrl } from "../transfer/transfer-out-util";
+import { EhrUrlNotFoundError } from "../../errors/errors";
 
 export const getEhrCoreFromRepo = async (nhsNumber, conversationId) => {
   const coreMessageUrl = await retrievePresignedUrlFromRepo(nhsNumber, conversationId);
   logInfo(`Successfully retrieved presigned URL`);
-  const ehrCore = await downloadEhrFromUrl(coreMessageUrl);
+  const ehrCore = await downloadFromUrl(coreMessageUrl);
   logInfo(`Successfully retrieved EHR`);
   return ehrCore;
 };
 
 const retrievePresignedUrlFromRepo = async (nhsNumber, conversationId) => {
-  const config = initializeConfig();
-  const repoUrl = `${config.ehrRepoServiceUrl}/patients/${nhsNumber}`;
+  const configObject = config();
+  const repoUrl = `${configObject.ehrRepoServiceUrl}/patients/${nhsNumber}`;

```

Comment:
```suggestion
  const configObject = config();
  const repoUrl = `${configObject.ehrRepoServiceUrl}/patients/${nhsNumber}`;

```
Is 'configObject' the best name to give this?